app/models/ticket_purchase.rb

```diff
@@ -2,6 +2,7 @@ class TicketPurchase < ActiveRecord::Base
   belongs_to :ticket
   belongs_to :user
   belongs_to :conference
+  belongs_to :payment
```

Comment:
> Please check it again. You should be able to use payment directly as @mdeniz says.

It was not working because one to many association between Payment and Ticket Purchase wasn't properly set up.